pandas/_libs/tslibs/nattype.pyx

```diff
@@ -547,113 +547,122 @@ class NaTType(_NaT):
         Parameters
         ----------
         tz : str or timezone object, default None
-            Timezone to localize to
+            Timezone to localize to.
         """)
     round = _make_nat_func('round',  # noqa:E128
         """
         Round the Timestamp to the specified resolution.
 
         Parameters
         ----------
-        freq : a freq string indicating the rounding resolution
-        ambiguous : bool, 'NaT', default 'raise'
-            - bool contains flags to determine if time is dst or not (note
-              that this flag is only applicable for ambiguous fall dst dates)
-            - 'NaT' will return NaT for an ambiguous time
-            - 'raise' will raise an AmbiguousTimeError for an ambiguous time
-
-            .. versionadded:: 0.24.0
-        nonexistent : 'shift_forward', 'shift_backward, 'NaT', timedelta, \
-default 'raise'
+        freq : str
+            Frequency string indicating the rounding resolution.
+        ambiguous : {'raise', 'NaT', bool}, default 'raise'
+            The behavior is as follows:
+
+            * bool contains flags to determine if time is dst or not (note
+              that this flag is only applicable for ambiguous fall dst dates).
+            * 'NaT' will return NaT for an ambiguous time.
+            * 'raise' will raise an AmbiguousTimeError for an ambiguous time.
+
+            .. versionadded:: 0.24.0.
+        nonexistent : {'raise', 'shift_forward', 'shift_backward, 'NaT', \
```

Comment:
Did you build the html to see if this renders ok?

Reply:
I checked for `pandas.Timestamp` and everything looked good. Couldn't figure out how to build it for just the `NaTType` class, but it should be fine considering its the same docstring as the Timestamp class.